nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/custom/Polygamma.java

```diff
@@ -0,0 +1,40 @@
+package org.nd4j.linalg.api.ops.custom;
+
+import lombok.NonNull;
+import org.nd4j.autodiff.samediff.SDVariable;
+import org.nd4j.autodiff.samediff.SameDiff;
+import org.nd4j.base.Preconditions;
+import org.nd4j.linalg.api.ndarray.INDArray;
+import org.nd4j.linalg.api.ops.DynamicCustomOp;
+
+public class Polygamma extends DynamicCustomOp {
+
+    public Polygamma() {
+        super();
+    }
+
+    public Polygamma(@NonNull INDArray n, @NonNull INDArray x) {
+        Preconditions.checkArgument(n.shape() != x.shape(),
+                "Polygamma: n and x must have the same shapes");
```

Comment:
FYI: other than %s (which is used for integers, strings, arrays, general objects) we have this:
https://github.com/eclipse/deeplearning4j/blob/master/nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/util/NDArrayPreconditionsFormat.java

So we can write things like:
```
Preconditions.checkArgument(n.shape() != x.shape(),
                "Polygamma: n and x must have the same shapes, got n.shape=%ndShape, x.shape=%ndShape", n, x);
```